examples/hello-world/hello-pt/README.md

```diff
@@ -30,3 +30,68 @@ $ ls /tmp/nvflare/simulate_job/
+
+When the global model is initialized on the FL server-side, it is then broadcasted to all the FL clients. 
+The clients can use this initial model as a starting point for training. 
+The advantage of server-side model initialization is that the model only needs to be initialized once in one location 
+(the server), and then distributed to all clients, ensuring that all clients have the same initial model. 
+However, it is important to note that server-side model initialization may present potential security risks 
+if custom Python code is run on the server. An alternative approach for server-side initialization is to use 
+a predefined model file as the initialization model. The ScatterAndGather controller is using persistor to reads / init model from server-side. 
+
+In this example, we are using **client-side** model initialization approach. 
+
+The client-side model initialization avoids server-side custom code as well as without extra setup in the model-file based approach.
+client side initialization asks every client to send the initialized model as a pre-task in the workflow, before the training starts.
+On the server side, once the server receive the initial models from clients, server can choose different strategies to leverage the models
+from different clients:
+* Select one model randomly among all clients’ models, then use it as the global initial model
+* Apply aggregation function to generate the global initial model
+
+In this example,we use _InitializeGlobalWeights_ controller, which have implemented the following strategies ( weight_method)
+* Weight_method = "first" , then use the weights reported from the first client;
+* weight_method = "client", then only use the weights reported from the specified client.
+
+If one’s use case demands a different strategy, then you can implement a new model initialization controller.
+
+Looking at the job workflow, we have defined three workflows in config_fed_server.json
+  * pre_train ( model initialization )  with _InitializeGlobalWeights_ controller
+  * scatter_and_gather (train) with _ScatterAndGather_ controller
+  * cross_site_validate (cross validation) with _CrossSiteModelEval_
+
+```
+ "workflows": [
+      {
+        "id": "pre_train",
+        "name": "InitializeGlobalWeights",
+        "args": {
+          "task_name": "get_weights"
+        }
+      },
+      {
+        "id": "scatter_and_gather",
+        "name": "ScatterAndGather",
+        "args": {
+            ... skip arguments ...
+        }
+      },
+      {
+        "id": "cross_site_validate",
+        "name": "CrossSiteModelEval",
+        "args": {
+          "model_locator_id": "model_locator"
+        }
+      }
+  ]
+```
+
+Once the global model is initialized, it is set to the fl_ctx as property and then pass to the 
+next controller (Scatter and Gather) in the training step. 
```

Comment:
Should we just make it `ScatterAndGather`?

One final note here, we said it upfront that `The ScatterAndGather controller is using persistor to reads / init model from server-side.`

The real scenario is `ScatterAndGather` will first try to use persistor to reads / init model from server-side. 

Then it will look for fl_ctx.get_prop(AppConstants.GLOBAL_WEIGHTS)